docs/Getting-Started-TPU-VM.md

```diff
@@ -2,7 +2,49 @@
 
 This guide will walk you through the steps to get started with Levanter on TPU VMs.
 
-## Google Cloud Setup
+## Overview
```

Comment:
This is a rather long document; to make it easier for a newcomer, would it be possible to organize it a bit more?  Start with the necessary steps: (1) installation, (2) launching jobs.  Then under (2), say there is a) running a one-off b) running using Ray.  That would make it easier to know how any individual piece fits into the larger puzzle.

Reply:
separate PR?

Reply:
Yeah, that's fine